donor-portal/src/pages/newsroom/index.tsx

```diff
@@ -57,33 +53,52 @@ export default function NewsRoom() {
         <div className="news-article-list">
           <h2> News </h2>
           <div className="news-article-list-item">
-            <strong>
-              Pension Fund Coalition for Inclusive Capitalism Releases Resource
-              to Focus Investments on Long-term Value Creation
-            </strong>
-            <span className="red">May 16, 2022</span>
-          </div>
-          <div className="news-article-list-item">
-            <strong>
-              BCI Receives 2021 Pension Leadership Award for Sustainable
-              Investing
-            </strong>
-            <span className="red">May 16, 2022</span>
-          </div>
-          <div className="news-article-list-item">
-            <strong>
-              BlackRock Subscribes to the Sustainable Development Goals Dataset
-              of the
-            </strong>
-            <span className="red">May 16, 2022</span>
+            <a
+              href="https://www.facebook.com/srilankaredcross/posts/pfbid0cC25DMKmdghvTXLhJBQxscS3buPkLuB3Fu5PLCPBDmmnuEQEZGoXFRdRDBfKuvnjl"
+              target="_blank"
+              rel="noreferrer"
+            >
+              <p>
+                <strong>
+                  July 29th 2022: The International Committee of the Red Cross
+                  has donated a consignment of 10,000 blood bags to the National
+                  Blood Bank.
+                </strong>
+              </p>
+            </a>
           </div>
           <div className="news-article-list-item">
-            <strong>Paul Finch Reappointed to Board of Directors </strong>
-            <span className="red">May 16, 2022</span>
+            <a
+              href="https://www.facebook.com/srilankaredcross/posts/pfbid02Lgtf7Zh6wLDjfgyBA53hPWxMDYKjR5TnbtDNY4RZD8sw6Rnd1Wj9HPfgCcfCzLTCl 
+              "
+              target="_blank"
+              rel="noreferrer"
+            >
+              <p>
+                <strong>
+                  June 9th 2022: Understanding the critical requirement of blood
+                  in Sri Lanka, Sri Lanka Red Cross Society - Colombo Branch
+                  organised a blood donation camp in Colombo, today.
+                </strong>
+              </p>
```

Comment:
Do we need the p tag? Doesn't it work like this?
```suggestion
                <strong>
                  June 9th 2022: Understanding the critical requirement of blood
                  in Sri Lanka, Sri Lanka Red Cross Society - Colombo Branch
                  organised a blood donation camp in Colombo, today.
                </strong>
```

Reply:
also, we need the date separately.